### DISCUSSION TITLE
eCapture Beginner's Guide

### DISCUSSION THREAD

**Discussion by cfc4n** [gojue/ecapture#114] [Ideas]
I want to put together a newbie guide document, who can help me. Now I have made a website [https://ecapture.cc](https://ecapture.cc) , repo : [https://github.com/ehids/ecapture.cc](https://github.com/ehids/ecapture.cc)

Anyone want to join in?

@vincentmli @nfsec @chriskaliX @cuishuang @xjas @xujiajiadexiaokeai @yihong0618

**Comment by vincentmli** [comment]
ecapture is simple to use for me, not sure if it requires user guide :) I myself suck at writing documents,  but I like recording hands-on video, for example use ecapture in container https://youtu.be/3trlo_eJVlk

**Reply by cfc4n** [reply]
Could you add a Makedown file `docker.md` like `android.md` with your video link into https://github.com/ehids/ecapture.cc/blob/master/src/zh/examples/ ,  and append some words.  Thanks.

**Reply by vincentmli** [reply]
will do

**Reply by vincentmli** [reply]
@cfc4n I send a PR https://github.com/ehids/ecapture.cc/pull/2 to link the video :)

**Comment by chriskaliX** [comment]
The guide in README doc looks fine to me. Maybe we could extract the guide part and add some use cases, then put it to the website

**Reply by cfc4n** [reply]
Sure, and I create Table of Contents of Documents. I think that you are good at `develop` phase...  so , will you do that?

**Reply by chriskaliX** [reply]
Yes, I'll try my best. :-)

**Reply by chriskaliX** [reply]
BTW, would you please share the `Table of Contents of Documents`, or we can discuss them personally.

**Reply by cfc4n** [reply]
in https://github.com/ehids/ecapture.cc/blob/master/src/zh/develop/index.md  , 
```text
# 编译
## 工具链 tool chain
## 操作系统
* 内核版本
* 系统参数
* 编译参数

# 自定义开发
## 项目结构
## 内核空间代码
## 用户空间代码
```

Each part has its own Markdown file.

**Reply by cfc4n** [reply]
except 
```
* 内核版本
* 系统参数
* 编译参数
```
